src/main/java/org/xstefank/webhook/WebHookEndpoint.java

```diff
@@ -20,7 +20,9 @@
     @Path("/pull-request")
     @Consumes(MediaType.APPLICATION_JSON)
     public void processPullRequest(JsonNode pullRequestPayload) {
-        templateChecker.checkPR(pullRequestPayload);
+        if (templateChecker.shouldSkip(pullRequestPayload)) {
```

Comment:
definitely supports my previous comment, it should not be required to call these methods (shouldSkip and checkPR) separately on the same object